code/modules/clothing/under/accessories.dm

```diff
@@ -378,3 +378,37 @@
 	new /obj/item/gun/ballistic/revolver/detective(src)
 	new /obj/item/ammo_box/c38(src)
 	new /obj/item/ammo_box/c38(src)
+
+/////////////////////
+//Armour Accessories//
+/////////////////////
+
+/obj/item/clothing/accessory/softpadding
+	name = "soft padding"
+	desc = "A protective padding meant to cushion the wearer from melee harm."
+	icon_state = "padding"
+	item_color = "nothing"
+	armor = list("melee" = 20, "bullet" = 10, "laser" = 0, "energy" = 0, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = -20, "acid" = 45)
```

Comment:
HOS jumpsuit: +10
Soft padding: +20 (30)
Ash Drake Armour: +70 (100)
You can get 100 melee armour on the body and limbs

Reply:
All you need is any armour with +70 melee resistance

Reply:
I'd argue this is more an issue with Lavaland's insane loot but to be fair I was just using the original PR's values, I can halve them.